src/test/java/org/broadinstitute/hellbender/tools/walkers/bqsr/ApplyWholeBQSRDataflowIntegrationTest.java

```diff
@@ -1,79 +0,0 @@
-package org.broadinstitute.hellbender.tools.walkers.bqsr;
-
-import org.broadinstitute.hellbender.CommandLineProgramTest;
-import org.broadinstitute.hellbender.exceptions.UserException;
-import org.broadinstitute.hellbender.tools.IntegrationTestSpec;
-import org.broadinstitute.hellbender.utils.test.ArgumentsBuilder;
-import org.broadinstitute.hellbender.utils.test.BaseTest;
-import org.testng.annotations.DataProvider;
-import org.testng.annotations.Test;
-
-import java.io.IOException;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.List;
-
-public final class ApplyWholeBQSRDataflowIntegrationTest extends CommandLineProgramTest {
```

Comment:
No. I also deleted ApplyWholeBQSRDataflow. It's not necessary anymore and keeping it would be an unnecessary maintenance burden.